scripts/scripts.js

```diff
@@ -197,6 +197,19 @@ export const fetchLangDatabyFileName = async (fileName) => {
     return [];
   }
 };
+export const fetchGlobalProperty = async (key) => {
```

Comment:
@skumawat1 We can use fetchPlaceholders() method from aem.js instead of this method.